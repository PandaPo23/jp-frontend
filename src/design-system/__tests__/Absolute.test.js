import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Absolute from '../Absolute';

describe('Absolute component', () => {
  it('should render as Absolute', () => {
    const wrapper = mount(<Absolute />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
