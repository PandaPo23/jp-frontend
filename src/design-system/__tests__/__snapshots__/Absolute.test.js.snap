// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Absolute component should render as Absolute 1`] = `
.emotion-0 {
  position: absolute;
  position: absolute;
}

<Absolute
  position="absolute"
>
  <div
    className="emotion-0"
  />
</Absolute>
`;
